rustic-doc.el

```diff
@@ -30,6 +30,12 @@
     (require 'xdg)
     (fset 'rustic-doc--xdg-data-home 'xdg-data-home)))
 
+(defconst rustic-doc-pandoc-min-version
+  (rustic-mode--make-semver :major 2 :minor 1))
```

Comment:
do you have local changes that you haven't committed to the PR branch? looks like you missed a rename here:

`rustic-mode--make-semver` should be `rustic-doc--make-semver`